src/Parser/Mage/Fire/Modules/Features/Cinderstorm.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import Wrapper from 'common/Wrapper';
+import { formatNumber, formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Analyzer from 'Parser/Core/Analyzer';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+
+const DAMAGE_WINDOW_MS = 5000;
+
+class Cinderstorm extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    abilityTracker: AbilityTracker,
+  };
+
+  cinderHits = 0;
+  totalHits = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.CINDERSTORM_TALENT.id);
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.CINDERSTORM_TALENT.id) {
+      return;
+    }
+    this.castTimestamp = event.timestamp;
+    this.cinderHits = 0;
+  }
+
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.CINDERSTORM_DAMAGE.id) {
+      return;
+    }
+    if (this.castTimestamp - event.timestamp < DAMAGE_WINDOW_MS && this.cinderHits < 6) {
```

Comment:
Why are you capping hits per cast at 6? When engaging multiple targets each cinder can strike more than once, allowing the possibility of well over 6 hits per cast. I assume this is to avoid showing an over 100% hit rate during AoE, but I think capping like this is a very misguided choice. In fact, my understanding is that the main reason one even specs for Cinderstorm is for AoE encounters. When I've played Fire in the past and specced Cinderstorm, the stat I was most interested in was how effectively I AoE'd with it.

Reply:
So originally i was only counting the cinders that hit the target, but Cinderstorm has no target. So i was going to use the Hit % for the statistic, but i didnt want a Hit % of over 100% cause that would just be weird, so i capped it at 6. But then I decided that Hits per Cast worked a lot better and wouldnt be awkward if the average was like 8 per cast ... but then i forgot to uncap the hits.